class RequestError(Exception):
    pass

class NotFoundError(Exception):
    pass